Dockerfile

```diff
@@ -44,77 +42,111 @@ RUN \
   gpg --batch --verify tomcat.tar.gz.asc tomcat.tar.gz && \
   tar -zxf tomcat.tar.gz -C /build/tomcat --strip-components=1 && \
   tar -zxf tcnative.tar.gz -C /build/tcnative --strip-components=1
+WORKDIR /build/tomcat
+# sh removes env vars it doesn't support (ones with periods)
+# https://github.com/docker-library/tomcat/issues/77
+RUN find ./bin/ -name '*.sh' -exec sed -ri 's|^#!/bin/sh$|#!/usr/bin/env bash|' '{}' +
+# Security improvements:
+# Remove server banner, Turn off loggin by the VersionLoggerListener, enable remoteIP valve so we know who we're talking to
```

Comment:
```suggestion
```

they are enumerated later, maybe we can still add a mention in the readme

Reply:
I'll just remove that. I don't want to introduce additional maintenance to keep the Dockerfile stage with the readme in sync.